web/profiles/joinup/src/Plugin/Block/AddContentBlock.php

```diff
@@ -86,7 +98,8 @@ public function build() {
       ];
 
       $solution_url = Url::fromRoute('solution.collection_solution.add', [
-        'rdf_entity' => $collection_contexts['collection']->getContextValue()->id(),
+        'rdf_entity' => $collection_contexts['collection']->getContextValue()
+          ->id(),
```

Comment:
Same here, either split or keep it on one line. I would prefer on one line, this is too trivial to split it over multiple lines.